homework1/task01_sample_project/calculator/__init__.py

```diff
@@ -1,5 +1,5 @@
 """
-This is confest.py docstring.
+This is __init__.py docstring.
 
 Some text to pass flake8.
 """
```

Comment:
Just remove `flake8-docstrings` `flake8-annotations` from requirements, pre-commit config, and uninstall them via pip.

Reply:
removed -> uninstalled

Reply:
Please remove this docstring.

Reply:
removed